src/main/java/fr/insee/rmes/api/CodeListsResources.java

```diff
@@ -28,8 +28,8 @@
 import io.swagger.v3.oas.annotations.tags.Tag;
 
 @RestController
-@RequestMapping(value="/codelist",produces = {"application/json"})
-@Tag(name = "Codelists", description = "Consultation Gestion API - Codelists")
+@RequestMapping(value="/",produces = {"application/json"})
+@Tag(name = "listeCode", description = "Consultation Gestion API - Codelists")
```

Comment:
https://github.com/InseeFr/Bauhaus-Back-Office/blob/b867d7173c56ba7cdd5d2fef74be3b661fbafb46/src/main/java/fr/insee/rmes/webservice/CodeListsResources.java#L29 
Endpoint URL seem a little different in Bauhaus ?

Reply:
effectivement mais si tu regardes dans consultation-gestion, c'est bien listesCodes (https://qfrmesgncslht01/bauhaus/swagger-ui/#/Consultation_Gestion)

Reply:
mais Florian parlait plutôt de l'url pour aller requêter le service donc je peux toujours modifier le titre de cette section en Codelists. D'ailleurs j'ai laissé  Consultation Gestion API - Codelists, il faudrait que je le modifie aussi

Reply:
Bon au final j'ai fait ça pour garder quand même le titre de la section en anglais:
@RequestMapping(value="/",produces = {"application/json"})
@Tag(name = "Codes lists", description = "Consultation Gestion API - listes de code")